src/main/java/reflection/utils/ReflectionUtils.java

```diff
@@ -61,64 +84,92 @@ public static Object setValueForPrivateField(Annotation annotation, String key,
         return oldValue;
     }
 
-    public static int getCalculatedValue(Object object) {
-        Class clazz = object.getClass();
-
-        Stream.of(getDeclaredMethods(object)).forEach(method -> {
-            if (method.isAnnotationPresent(Invoke.class) && method.getAnnotation(Invoke.class).state()) {
-                try {
-                    method.invoke(object);
-                } catch (IllegalAccessException | InvocationTargetException e) {
-                    e.printStackTrace();
-                }
-            }
-        });
-//        for (Method method : methods) {
-//            if (method.isAnnotationPresent(Invoke.class) && method.getAnnotation(Invoke.class).flag()) {
-//                method.invoke(object);
-//            }
-//        }
-
-        Object declaredField = null;
+    private static Method[] getDeclaredMethods(final Object object) {
+        return object.getClass().getDeclaredMethods();
+    }
 
+    public static Method getDeclaredMethod(final Object object, final String name) {
+        Method method = null;
         try {
-            Field field = clazz.getDeclaredField("sum");
+            method = object.getClass().getDeclaredMethod(name);
+        } catch (NoSuchMethodException e) {
+            e.printStackTrace();
+        }
+        return method;
+    }
+
+    private static Object getAnnotationValue(final String name, final Class<? extends Annotation> annotation) {
+        Object value = null;
+        try {
+            value = annotation.getDeclaredMethod(name).getDefaultValue();
+        } catch (ReflectiveOperationException e) {
+            e.printStackTrace();
+        }
+        return value;
+    }
+
+    public static void setIntPrivateField(final Object object, final String nameField, final int fieldValue) {
+
+        try {
+            Field field = object.getClass().getDeclaredField(nameField);
+            boolean flag = field.isAccessible();
+
+            field.setAccessible(true);
+            field.set(object, fieldValue);
+            field.setAccessible(flag);
+        } catch (ReflectiveOperationException e) {
+            e.printStackTrace();
+        }
+    }
+
+    public static Object getValueFromPrivateField(final Object object, final String nameFieldWithAnn) {
```

Comment:
What is "nameFieldWithAnn"
It looks like common method to get value from any object by any name

getPrivateFieldValue(Object object, String fieldName)